a/build.gradle

```diff
@@ -1,23 +1,5 @@
 plugins {
-    id 'scala'
     id 'com.github.prokod.gradle-crossbuild'
 }
 
 apply from: "$rootDir/gradle/scala_211_212.gradle"
-
-// Declare the variants as "outgoing" artifacts
-configurations {
```

Comment:
I don't fully understand why you removed that. The use case is the following:

1. A downstream project wants to explicitly use the Scala variant 2.11 or 2.12, not just the default...whatever it is.
2. A downstream project wants to use both variants, 2.11 _and_ 2.12. Just depending on the `default` configuration won't do.

Am I missing something?

Reply:
I removed this because the plugin is smart enough to do this for you :) - at least should be :D
Regarding `default` configuration this is just because of test purposes. when you do `./gradlew .clean buiild ....` test is being triggered and this fails because the `default` scala was not aligned.

Be careful! this does not mean that the project did not cross compile correctly.
I suggest to see also `./gradlew tasks` to see available tasks, specifically for crossbuilding